README.rst

```diff
@@ -1,6 +1,12 @@
-=====================
-django-split-settings
-=====================
+.. raw:: html
+
+    <a href="https://github.com/sobolevn/django-split-settings">
+      <p align="center">
+         <img src="https://i.imgur.com/vISkpwj.png">
+      </p>
+    </a>
+
+----------
```

Comment:
Now, when i see how its looks i'm not sure that is line is cool. Should I delete it?